easybuild/easyconfigs/g/gsutil/gsutil-5.10-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,117 @@
+
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Center - Seattle Washington - US
+#
+easyblock = 'PythonBundle'
+
+name = 'gsutil'
+version = '5.10'
+
+homepage = 'https://cloud.google.com/storage/docs/gsutil'
+description = """gsutil is a Python application that lets you access Cloud Storage from the command line."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+builddependencies = [
+    ('buildutils', '2.37'),
```

Comment:
You made a typo and wrote `buildutils`  instead of `binutils`. That's where the CI failure comes from.

Reply:
Ok, fixed it myself